### ISSUE TITLE
Feature: hotlink production rates

### ISSUE THREAD

**Issue by scottj97** [frosty656/frosty656.github.io#23]
I'd like to be able to click on a number and have the display change to show me the production tree for that one item.

For example I'm viewing 1.4148 Earth Tokens. In the Summary View is:
```
Matter Compressor: 14.15 (3 Manufacturer)
```

I'd like to be able to click on the "14.15" and have the display change to show me the production tree/summary for only that one item, i.e. change the Items/Min box to 14.15 and the item selected to Matter Compressor.

This is useful when building my factory because I will often focus on one item at a time, building all of its inputs, with the ultimate goal of maximizing Earth Token production.

**Comment by frosty656** [comment]
That is a great idea! Just to clarify your ask you would like to bring up an items tree by clicking on it in the tree view of another item? Instead of the back button I could create a breadcrumb trail that would track all of clicked tree items to make it easy to navigate backwards. If you want to be able to go back and forth instead of having to drill down I could do tabs so that you can easily switch between views. Would love you hear your feedback!

**Comment by scottj97** [comment]
Yes, exactly...though not necessarily in the tree view. In the summary view also. Typically my "top level" is the summary view for maximum Earth Tokens, and then I want to drill down into the tree view for one specific item.

For example I'll start with the maximum Earth Tokens summary, then drill down into Heat Sinks. I'll go build just enough extractors/furnaces/workshops to make the perfect number of Heat Sinks needed for the entire world. Then I go back up to the maximum Earth Tokens summary and choose another item to work on.

**Comment by frosty656** [comment]
That's seems like a great feature to add I will work on adding that.
